sources/packages/api/src/types/pdStatus.ts

```diff
@@ -0,0 +1,9 @@
+/**
+ * Enumeration for student permanent disability status.
+ */
+export enum StudentPDStatus {
+  Yes = "Yes",
+  No = "No",
+  Not_Requested = "Not Requested",
```

Comment:
Please rename it to "NotRequested". We are not supposed to be using "_" in this case. If the application has some other place using it, it is probably wrong.